File: #21/days.php
<?php
// Web APIとして使用するスクリプトです

// 入力値が無い場合を鑑み初期値を設定
$i = 'saturday';

// URLから"word"に対応するデータがあれば、$iに代入
if(array_key_exists('word', $_GET)) {
	$i = $_GET['word'];
}

// 文字列比較のため、すべて小文字に変換しなおす
$i = strtolower($i);

// $iの文字列により返却するjsonデータを決定
switch ($i) {
	case 'sunday':
		$str = '{"sunday" : "日曜"}';
		break;

	case 'monday':
		$str = '{"monday" : "月曜"}';
		break;

	case 'tuesday':
		$str = '{"tuesday" : "火曜"}';
		break;

	case 'wednesday':
		$str = '{"wednesday" : "水曜"}';
		break;

	case 'thursday':
		$str = '{"thursday" : "木曜"}';
		break;

	case 'friday':
		$str = '{"friday" : "金曜"}';
		break;

	default:
		$str = '{"saturday" : "土曜"}';
		break;

}

// jsonとして結果を出力(=レスポンスとして返す)
print "$str";

?>

File: #22/days.py
#! /usr/local/bin/python3
import io, sys, cgi

sys.stdout = io.TextIOWrapper(sys.stdout.buffer, encoding='utf-8')
sys.stderr = io.TextIOWrapper(sys.stderr.buffer, encoding='utf-8')


query = cgi.FieldStorage()

word = 'saturday'

# Getデータを取得する
if 'word' in query:
    word = query['word'].value


# 文字列比較のため小文字にする
word = word.lower()

if word == 'sunday':
    result = '{"sunday" : "日曜"}'
elif word == 'monday':
    result = '{"monday" : "月曜"}'
elif word == 'tuesday':
    result = '{"tuesday" : "火曜"}'
elif word == 'wednesday':
    result = '{"wednesday" : "水曜"}'
elif word == 'thursday':
    result = '{"thursday" : "木曜"}'
elif word == 'friday':
    result = '{"friday" : "金曜"}'
else:
    result = '{"saturday" : "土曜"}'


print('Content-type: text/html; charset=UTF-8')
print()
print(result)